src/codeCache.h

```diff
@@ -98,10 +100,11 @@ class NativeCodeCache : public CodeCache {
 
     void add(const void* start, int length, const char* name, bool update_bounds = false);
     void sort();
-    const char* binarySearch(const void* address);
+    const char* binarySearch(const void* address, LinearAllocator* allocator);
     const void* findSymbol(const char* name);
     const void* findSymbolByPrefix(const char* prefix);
     const void* findSymbolByPrefix(const char* prefix, int prefix_len);
+    void setBuildId(const char *build_id, int build_id_len);
```

Comment:
```suggestion
    void setBuildId(const char* build_id, int build_id_len);
```
If you are interested in keeping conventions.

There are some other cases of this.

Reply:
Yeah, I tried to keep but seem to have forgotten it here :)